### PR TITLE
[162, 163] Fix build and test hooks (main)

### PR DESCRIPTION
**Pull request by FifthPotato**
Do not install unittest-xml-reporting==1.14.0
Update versions of externals packages in build hook

### CODE REVIEW

**Review by korydraughn — COMMENTED** [review]
Looks good to me.

Will approve after you've confirmed things are working.

**Review by alanking — COMMENTED** [review]
Agreed

**Comment by FifthPotato** [comment]
Tests pass.

**Comment by korydraughn** [comment]
Running tests for EL8 and EL9. If those succeed, then this will be ready for merging.

will report back once those tests complete.

**Comment by korydraughn** [comment]
Just realized this is for the main branch.

We need this to happen on 4-3-stable.

Postponing testing until that's done.

**Comment by alanking** [comment]
@FifthPotato - Let's go ahead and cherry-pick this to 4-3-stable and then we'll merge this after or at around the same time.

File: irods_consortium_continuous_integration_build_hook.py [hunk 1]
```diff
@@ -15,9 +15,9 @@ def add_cmake_to_front_of_path():
 
 def install_building_dependencies(externals_directory):
     externals_list = [
-        'irods-externals-boost1.81.0-0',
-        'irods-externals-clang-runtime13.0.0-0',
-        'irods-externals-clang13.0.0-0',
+        'irods-externals-boost-libcxx1.81.0-1',
```
Comment (korydraughn) [inline irods_consortium_continuous_integration_build_hook.py:18]: @SwooshyCueb Is this the correct externals pkg for main?

Looking at the following lines from irods/irods.
- https://github.com/irods/irods/blob/bc2a9bf0d8d3587d1aad8bf6a9b3db650e0d6944/CMakeLists.txt#L25-L43

Comment (SwooshyCueb) [reply irods_consortium_continuous_integration_build_hook.py:18]: No, main should have non-libcxx packages, and should omit clang-runtime.

**Review by SwooshyCueb — CHANGES_REQUESTED** [review]

**Comment by korydraughn** [comment]
@FifthPotato Please address the review comments and leave a comment once completed.

**Comment by FifthPotato** [comment]
Updated. Tests still pass.

**Review by alanking — APPROVED** [review]

**Comment by korydraughn** [comment]
For future PRs, let's keep spaces out of the issue number list (in square brackets) for commit messages. For example, commit messages should look like the following:
```
[#162,#163] <message>
```

**Pull request merged**